src/util/ue2_graph.h

```diff
@@ -796,6 +796,9 @@ class ue2_graph : graph_detail::graph_base {
         explicit prop_map(value_type P_of::*m_in) : member(m_in) { }
 
         reference operator[](key_type k) const {
+            if (k.raw() == nullptr || &(k.raw()->props) == nullptr) {
+                throw std::invalid_argument("Invalid key");
+            }
```

Comment:
I wonder if this is going to drop performance.

Reply:
cppcheck thinks that this is a redundant check so im gonna suppress this